src/Microsoft.DotNet.Interactive.Formatting/HtmlFormatter.cs

```diff
@@ -121,7 +128,7 @@ void Initialize()
 
         new HtmlFormatter<string>((s, context) =>
         {
-            // If PlainTextPreformat is true, then strings
+            // If PreformatPlainText is true, then strings
```

Comment:
Is `PreformatPlainText` referring to an option that is present inside `context`?

Reply:
Deleting this comment. It was outdated and refers to something that no longer exists.